articles/mysql/flexible-server/concepts-high-availability.md

```diff
@@ -184,7 +184,8 @@ Data-in Replication isn't supported for HA servers. But Data-in Replication for
     1. Enable HA.
 
 - **To reduce downtime, can I fail over to the standby server during server restarts or while scaling up or down?** </br>
-Currently, when you do a scale up or scale down operation, the standby and primary are scaled at the same time. So failing over doesn't help. Allowing scaling up of the standby first, followed by failover, and then scaling up of the primary is on our roadmap but isn't supported yet.</br>
+Currently, Azure MySQL Flexible Server has utlized Planned Failover to optmize the HA operations including scaling up/down, and planned maintenance to help reduce the downtime.
+When such operations started, it would operate on the original standby instance first, and followed by triggering a planned failover operation, and then operate on the original primary instance. </br>
```

Comment:
```suggestion
When such operations started, it would operate on the original standby instance first, followed by triggering a planned failover operation, and then operate on the original primary instance. </br>
```